scalableminds/webknossos
# Fix initial loading of data for last zoomstep (#2962)

Trigger the prefetch strategies once initially.

### URL of deployed dev instance (used for testing):
- https://fixinitialloading.webknossos.xyz

### Steps to test:
- Zoom out as much as possible, then save and reload the page. Data should be rendered without moving.

### Issues:
- fixes #2962 

------
